docs/calico-with-docker/docker-network-plugin/CalicoAsService.md

```diff
@@ -0,0 +1,141 @@
+<!--- master only -->
+> ![warning](../../images/warning.png) This document applies to the HEAD of the calico-containers source tree.
+>
+> View the calico-containers documentation for the latest release [here](https://github.com/projectcalico/calico-containers/blob/v0.17.0/README.md).
+<!--- else
+> You are viewing the calico-containers documentation for release **release**.
+<!--- end of master only -->
+
+# Running Calico Node Containers as a Service
+
+This guide explains how to run Calico as a system process or service. We
+include examples for Systemd, but the commands can be applied to other init
+daemons such as upstart.
+
+## Running the Calico Node Container as a Systemd Service
+The `calicoctl node` command already starts calico as its own container process
+on your system. There is no need to run the base `calico/node` container with a
```

Comment:
Wording is still a little odd here - it's not really saying what you need to do.

For example, include the --detach flag so that it blocks.  Could also add a short one-liner that detach mode is not possible when running with the --libnetwork - see details below about blah blah.  That ties the two sections together.